packages/kit/src/runtime/server/page/render.js

```diff
@@ -304,6 +309,15 @@ export async function render_response({
 		etag: `"${hash(html)}"`
 	});
 
+	if (link_header_preload.size) {
+		headers.set(
+			'link',
+			Array.from(link_header_preload)
+				.map((href) => `<${href}>; rel=preload`)
+				.join(',')
+		);
+	}
+
```

Comment:
I'm pretty sure this is incorrect — modules should have `rel=modulepreload`, not `rel=preload`

Reply:
Ya you're right 😅 
- https://datatracker.ietf.org/doc/html/rfc8288#section-2.1.1.1
- https://www.iana.org/assignments/link-relations

fixed. <3